docs/DeveloperGuide.adoc

```diff
@@ -292,20 +292,66 @@ Refer to the guide <<DevOps#, here>>.
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |Forgetful user |Write down the day's events | Remember what happened in that day
 
-|`* * *` |user |add a new person |
+|`* * *` |user |Have a diary  | Write my thoughts and feelings for that day
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |Unthrifty traveller |Notified if my spending goes beyond my planned levels |Adjust my budget
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |Traveller |Get my expenses to automatically converted to SGD |Avoid manual currency conversion
+
+|`* * *` |Budget traveller |Set an individual budget for each day |Minimise my spending
+
+|`* * *` |Traveller |To see amount spent each day/trip and balance left for each day/trip |Record my spending
+
+|`* * *` |Forgetful user |Make a checklist for items to bring |Pack without forgetting anything
+
+|`* * *` |Careless traveller |Set the quantity of items I need to pack |Pack without missing out anything
+
+|`* * *` |Lazy traveller |The list to be auto generated/ pre-set packing list |So that I dont have to put in so much effort
+
+|`* * *` |Traveller |Maintain a list of accomodations |Better plan my accomodations during the trips
+
+|`* * *` |Forgetful traveller |Keep track of my transportation information |Avoid being late
+
+|`* * *` |Organised traveller |Plan my trip in terms of days |Split my plan into suitable chunks
+
+|`* * *` |Inexperienced planner |Know what did I miss out from my travel plan |Be reminded and plan for it
+
+|`* * *` |Organised traveller |Add activities to my daily itienerary |Plan for my trip
+
+|`* *` |User concerned with privacy |Want to private my diary entry|Minimize chance of someone else seeing them by accident
+
+|`* *` |User who likes pictures |Have a photo diary |Capture memories through pictures in a organised manner
+
+|`* *` |Careless Traveller |Be notified if I have forgetten certain aspects of my trip |Avoid being underprepared for my trip
+
+|`* *` |Cautious Traveller |Be notified if I miss out any dates i did not plan on accommodation |Have a place to sleep every night
+
+|`* *` |Traveller |Be able to print itinenary |Bring it around in my travels if my battery dies
+
+|`* *` |Traveller |Be able to undo my actions |Undo my mistakes
+
+|`* *` |Traveller |View all the information easily|
+
+|`* *` |User |Have a help guide|
+
+|`*` |Non Math inclined traveller |Get constant recommendations to my budget plan |Avoid having to do any difficult calculations
+
+|`*` |Undisciplined individual |Keep track of my expenses |Avoid spending over my budget
+
+|`*` |Command Line enthusiast |Tab autocomplete my commands |Be more efficient
+
+|`*` |User|Not have to navigate the UI frequently|
+
+|`*` |change my application into different themes|Visualise my routes on a map |make my application interface look better
```

Comment:
This might have been copied wrongly. Suggestion:

`User who likes customization` | `Change my application into different colour themes` | `Make my application's GUI looks better`
-- | -- | --